From 0923304c8561afb984dd90e4c2065865fea8b2aa Mon Sep 17 00:00:00 2001
From: Vyacheslav Bocharov <adeep@lexina.in>
Date: Mon, 7 Nov 2022 16:19:08 +0300
Subject: [PATCH 729/823] RADXA: arm64: amlogic: dts: meson: update meson-axg
 device-tree for new core, tx, rx phase clock settings.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 arch/arm64/boot/dts/amlogic/meson-axg.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-axg.dtsi b/arch/arm64/boot/dts/amlogic/meson-axg.dtsi
index f1f53fd98ae25..d40c14d910f54 100644
--- a/arch/arm64/boot/dts/amlogic/meson-axg.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-axg.dtsi
@@ -13,6 +13,7 @@
 #include <dt-bindings/reset/amlogic,meson-axg-audio-arb.h>
 #include <dt-bindings/reset/amlogic,meson-axg-reset.h>
 #include <dt-bindings/power/meson-axg-power.h>
+#include <dt-bindings/mmc/meson-gx-mmc.h>
 
 / {
 	compatible = "amlogic,meson-axg";
@@ -1969,6 +1970,8 @@ sd_emmc_b: mmc@5000 {
 
 				assigned-clocks = <&clkc CLKID_SD_EMMC_B_CLK0>;
 				assigned-clock-rates = <24000000>;
+
+				amlogic,mmc-phase = <CLK_PHASE_270 CLK_PHASE_0 CLK_PHASE_0>;
 			};
 
 			sd_emmc_c: mmc@7000 {
@@ -1984,6 +1987,8 @@ sd_emmc_c: mmc@7000 {
 
 				assigned-clocks = <&clkc CLKID_SD_EMMC_C_CLK0>;
 				assigned-clock-rates = <24000000>;
+
+				amlogic,mmc-phase = <CLK_PHASE_270 CLK_PHASE_0 CLK_PHASE_0>;
 			};
 
 			nfc: nand-controller@7800 {
-- 
2.52.0

